frontend/app/components/ui/interactive/index.ts

```
@@ -0,0 +1,17 @@
+export type { ButtonProps, ButtonVariant, ButtonSize } from './Button';
+
+// Icon button component
+export { IconButton } from './IconButton';
+export type { IconButtonProps, IconButtonSize } from './IconButton';
+
+// Link component with states
+export { Link } from './Link';
+export type { LinkProps, LinkVariant, LinkSize } from './Link';
+
+// Placeholder - components will be implemented in subsequent tasks
+export {}; 
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This export makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)